pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -400,11 +400,10 @@ func (m *kubeGenericRuntimeManager) getPodContainerStatuses(uid kubetypes.UID, n
 			RestartCount: annotatedInfo.RestartCount,
 			State:        toKubeContainerState(c.State),
 			CreatedAt:    time.Unix(0, status.CreatedAt),
+			StartedAt:    time.Unix(0, status.StartedAt),
 		}
 
-		if c.State == runtimeapi.ContainerState_CONTAINER_RUNNING {
-			cStatus.StartedAt = time.Unix(0, status.StartedAt)
-		} else {
+		if c.State == runtimeapi.ContainerState_CONTAINER_EXITED {
```

Comment:
FYI, I attempted to write a simple unit test, but that'd requires quite a bit of refactoring